azure-pipelines.yml

```diff
@@ -155,19 +161,19 @@ jobs:
           # isDraft: true
           # isPreRelease: true
           # addChangeLog: false
-      # - task: DotNetCoreCLI@2
-      #   displayName: 'Push NuGet packages to myget.org'
-      #   inputs:
-      #     command: push
-      #     nuGetFeedType: external
-      #     packagesToPush: ${{ format('{0}/nuget/*.nupkg;!{0}/nuget/*.symbols.nupkg', parameters.Artifacts) }}
-      #     publishFeedCredentials: myget.org
-      #   condition: and(succeeded(), or(startsWith(variables['Build.SourceBranch'], 'refs/tags/v'), eq(variables['Build.SourceBranch'], 'refs/heads/master')))
-      # - task: DotNetCoreCLI@2
-      #   displayName: 'Push NuGet packages to nuget.org'
-      #   inputs:
-      #     command: push
-      #     nuGetFeedType: external
-      #     packagesToPush: ${{ format('{0}/nuget/*.nupkg;!{0}/nuget/*.symbols.nupkg', parameters.Artifacts) }}
-      #     publishFeedCredentials: nuget.org
-      #   condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/tags/v'))
+      - task: DotNetCoreCLI@2
+        displayName: 'Push NuGet packages to myget.org'
+        inputs:
+          command: push
+          nuGetFeedType: external
+          packagesToPush: $(System.ArtifactsDirectory)/nuget/**/*.nupkg;!$(System.ArtifactsDirectory)/nuget/**/*.symbols.nupkg'
+          publishFeedCredentials: myget.org
+        condition: and(succeeded(), or(startsWith(variables['Build.SourceBranch'], 'refs/tags/v'), eq(variables['Build.SourceBranch'], 'refs/heads/master')))
+      - task: DotNetCoreCLI@2
+        displayName: 'Push NuGet packages to nuget.org'
```

Comment:
unfortunately at the moment we can't really push to nuget.org because we have dependencies on private feeds https://github.com/OmniSharp/omnisharp-roslyn/blob/master/NuGet.Config

Reply:
why not?  They just need to have the private feed to use the package restore... 😈 

Thoughts on what to do?  Just not publish there yet?

Reply:
I would start by just publishing to myget and not to nuget yet.
I am not so convinced that the API is in the best shape yet so starting with myget only sounds like a good way to "ease in" if you ask me.